requirements.txt

```diff
@@ -1,5 +1,11 @@
-django==1.8
-djangorestframework==2.3.11
-django-nose==1.4
+Django==1.8
 django-hosts==1.2
-unittest2==1.1.0
+django-nose==1.4
+djangorestframework==2.3.11
+linecache2==1.0.0
+nose==1.3.7
+python-dateutil==2.6.0
+PyYAML==3.12
+six==1.10.0
+traceback2==1.4.0
+unittest2==1.1.0
```

Comment:
Yeah you're right, I removed some. We use `dateutil` in `test_cases.py`, and `six` (a package apparently used to ease Python 2/3 compatibility) is a dependency.